dashboard/test/controllers/channels_api_controller_test.rb

```diff
@@ -0,0 +1,95 @@
+require 'test_helper'
+
+class ChannelsApiControllerTest < ActionController::TestCase
+  include Devise::Test::ControllerHelpers
+
+  setup do
+    @storage = create(:project_storage)
+    @storage_id = @storage.id
+    @projects = Projects.new(@storage_id)
```

Comment:
In the past I've had to put project setup in a `setup_all` block -- I wonder if that might help your test failure?

Reply:
(I don't quite know why this happens -- a weird side effect from when we moved the projects table to dashboard)

Reply:
tried a few variations of moving everything / just the parts i suspect to `setup_all`, with the same results 😞